### PR TITLE
Add compat data for Node (just properties to start)

### PR DESCRIPTION
**Pull request by maboa**
https://developer.mozilla.org/en-US/docs/Web/API/Node

Note - starting off with just the properties, will add methods once this has been approved.

Also note there are a few properties with no table and so I have represented this as all nulls, can't remember if this is the correct procedure.

### CODE REVIEW

File: api/Node.json [hunk 1]
```diff
@@ -0,0 +1,1107 @@
...
+          "webview_android": {
+            "version_added": true,
+            "notes": "WebKit and old versions of Blink incorrectly do not make <code>Node</code> inherit from <code>EventTarget</code>."
+          },
+          "chrome": {
+            "version_added": true,
+            "notes": "WebKit and old versions of Blink incorrectly do not make <code>Node</code> inherit from <code>EventTarget</code>."
+          },
+          "chrome_android": {
+            "version_added": null
```
Comment (jpmedley) [inline api/Node.json]: Chrome Android should match Webview Android.

File: api/Node.json [hunk 2]
```diff
@@ -0,0 +1,1107 @@
...
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Node/baseURI",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": null
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/Node.json]: All Chrome versions should be `true`. You can verify support of all of Node's members [in the IDL file](https://cs.chromium.org/chromium/src/third_party/WebKit/Source/core/dom/Node.idl?q=baseURI+file:%5Esrc/third_party/WebKit/Source/+package:%5Echromium$&dr=C&l=39).

File: api/Node.json [hunk 3]
```diff
@@ -0,0 +1,1107 @@
...
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Node/childNodes",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": "1"
+            },
+            "chrome_android": {
+              "version_added": "1"
```
Comment (jpmedley) [inline api/Node.json]: There was no version 1 for Chrome Android. Please change to `true`. Change Webview Android to true as well.

File: api/Node.json [hunk 4]
```diff
@@ -0,0 +1,1107 @@
...
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Node/firstChild",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": "1"
+            },
+            "chrome_android": {
+              "version_added": "1"
```
Comment (jpmedley) [inline api/Node.json]: Same as for `childNodes`.

File: api/Node.json [hunk 5]
```diff
@@ -0,0 +1,1107 @@
...
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": "4"
+            },
+            "chrome_android": {
+              "version_added": "2.3",
+              "notes": "probably earlier"
```
Comment (jpmedley) [inline api/Node.json]: Change this to `true` and remove the notes. Android Chrome and Android Webview follow Chrome version numbers, not Android.

File: api/Node.json [hunk 6]
```diff
@@ -0,0 +1,1107 @@
...
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Node/lastChild",
+          "support": {
+            "webview_android": {
+              "version_added": true
+            },
+            "chrome": {
+              "version_added": "1"
+            },
+            "chrome_android": {
+              "version_added": "1"
```
Comment (jpmedley) [inline api/Node.json]: Change to `true`.

File: api/Node.json [hunk 7]
```diff
@@ -0,0 +1,1107 @@
...
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Node/localName",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": true,
+              "version_removed": "46",
+              "notes": "This API was moved to the <code>Element</code> and <code>Attr</code> APIs according to the DOM4 standard."
```
Comment (jpmedley) [inline api/Node.json]: Copy all this to Webview and Chrome Android.

File: api/Node.json [hunk 8]
```diff
@@ -0,0 +1,1107 @@
...
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Node/namespaceURI",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": true,
+              "version_removed": "46",
+              "notes": "This API was moved to the <code>Element</code> and <code>Attr</code> APIs according to the DOM4 standard."
```
Comment (jpmedley) [inline api/Node.json]: Copy to Webview and Chrome Android

File: api/Node.json [hunk 9]
```diff
@@ -0,0 +1,1107 @@
...
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Node/nextSibling",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": null
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/Node.json]: This should be `true` for all 3 and both versions of Opera.

File: api/Node.json [hunk 10]
```diff
@@ -0,0 +1,1107 @@
...
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Node/nodeName",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": null
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/Node.json]: True for all Chromium browsers and Webview.

File: api/Node.json [hunk 11]
```diff
@@ -0,0 +1,1107 @@
...
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Node/nodePrincipal",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": true,
+              "version_removed": "46",
+              "notes": "This API was moved to the <code>Element</code> and <code>Attr</code> APIs according to the DOM4 standard."
```
Comment (jpmedley) [inline api/Node.json]: Copy to webview and Chrome Android.

File: api/Node.json [hunk 12]
```diff
@@ -0,0 +1,1107 @@
...
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Node/nodeType",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": null
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/Node.json]: True for all 5 Chrome-based.

File: api/Node.json [hunk 13]
```diff
@@ -0,0 +1,1107 @@
...
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Node/nodeValue",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": null
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/Node.json]: True for all 5.

File: api/Node.json [hunk 14]
```diff
@@ -0,0 +1,1107 @@
...
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Node/outerText",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": null
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/Node.json]: `false` for all 5.

File: api/Node.json [hunk 15]
```diff
@@ -0,0 +1,1107 @@
...
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Node/ownerDocument",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/Node.json]: `true` for all 5.

File: api/Node.json [hunk 16]
```diff
@@ -0,0 +1,1107 @@
...
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Node/parentElement",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/Node.json]: `true` for all 5.

File: api/Node.json [hunk 17]
```diff
@@ -0,0 +1,1107 @@
...
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Node/prefix",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/Node.json]: This is actually false for all 5.

File: api/Node.json [hunk 18]
```diff
@@ -0,0 +1,1107 @@
...
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Node/previousSibling",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": null
+            },
+            "chrome_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/Node.json]: `true` for all 5.

File: api/Node.json [hunk 19]
```diff
@@ -0,0 +1,1107 @@
...
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Node/rootNode",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": false,
+              "alternative_name": "getRootNode"
+            },
```
Comment (jpmedley) [inline api/Node.json]: This applies to all 5.

File: api/Node.json [hunk 20]
```diff
@@ -0,0 +1,1107 @@
...
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/Node/textContent",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": "1"
+            },
+            "chrome_android": {
+              "version_added": "4.4"
```
Comment (jpmedley) [inline api/Node.json]: Should be `true` for Webview, Chrome Android, and Opera.

File: api/Node.json (hunk 1 above)
Comment (maboa) [reply api/Node.json]: 👍

File: api/Node.json (hunk 2 above)
Comment (maboa) [reply api/Node.json]: 👍

File: api/Node.json (hunk 3 above)
Comment (maboa) [reply api/Node.json]: 👍

File: api/Node.json (hunk 4 above)
Comment (maboa) [reply api/Node.json]: 👍

File: api/Node.json (hunk 5 above)
Comment (maboa) [reply api/Node.json]: 👍 Assuming `webview_android` is `true` too.

File: api/Node.json (hunk 6 above)
Comment (maboa) [reply api/Node.json]: 👍

File: api/Node.json (hunk 7 above)
Comment (maboa) [reply api/Node.json]: 👍

File: api/Node.json (hunk 8 above)
Comment (maboa) [reply api/Node.json]: 👍

File: api/Node.json (hunk 9 above)
Comment (maboa) [reply api/Node.json]: 👍

File: api/Node.json (hunk 10 above)
Comment (maboa) [reply api/Node.json]: 👍

File: api/Node.json (hunk 11 above)
Comment (maboa) [reply api/Node.json]: 👍

Comment (maboa) [reply api/Node.json]: What about Opera here @jpmedley ?

File: api/Node.json (hunk 12 above)
Comment (maboa) [reply api/Node.json]: 👍

File: api/Node.json (hunk 13 above)
Comment (maboa) [reply api/Node.json]: 👍

File: api/Node.json (hunk 14 above)
Comment (maboa) [reply api/Node.json]: 👍

File: api/Node.json (hunk 15 above)
Comment (maboa) [reply api/Node.json]: 👍

File: api/Node.json (hunk 16 above)
Comment (maboa) [reply api/Node.json]: 👍

File: api/Node.json (hunk 17 above)
Comment (maboa) [reply api/Node.json]: 👍

File: api/Node.json (hunk 18 above)
Comment (maboa) [reply api/Node.json]: 👍

File: api/Node.json (hunk 19 above)
Comment (maboa) [reply api/Node.json]: 👍

File: api/Node.json (hunk 20 above)
Comment (maboa) [reply api/Node.json]: 👍

**Comment by maboa** [comment]
Should be updated :) - will add the methods once @Elchi3 has commented.

File: api/Node.json (hunk 11 above)
Comment (jpmedley) [reply api/Node.json]: I wasn't able to discover anything. `null` will display as a question mark.

**Pull request merged**